backend/src/routes/blog.ts

```diff
@@ -213,4 +213,42 @@ blogRouter.delete("/:id", async (c) => {
       message: "Error while deleting post",
     });
   }
-});
\ No newline at end of file
+});
+
+/**
+ * Retrieves all the blogs for the user in bulk
+ */
+blogRouter.get("/bulkUser/:id", async (c) => {
```

Comment:
Instead of creating a new route here, we can reuse existing `/blog/bulk` route with filter conditions as query parameters.